css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,50 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-fit {
```

Comment:
This rule is for a class named `column-fit`, but the class name you used in your HTML is `column-fith`. That means that this ruleset isn't being used at all. Also, it should be spelled `column-fifth`.